### ISSUE TITLE
PDF highlight annotation doesn't work

### ISSUE THREAD

**Issue by DRBragg** [Submitty/Submitty#5688]
**Describe the bug**
When selecting the highlighter tool you are unable to highlight the text

**Expected behavior**
When selecting the highlighter tool you are able to highlight the text

**To Reproduce**
1. clone and/or open the [Submitty/pdf-annotate.js](https://github.com/Submitty/pdf-annotate.js) dir
2. run the example
3. select the highlighter tool and attempt to highlight the text
4. See error (unable to select/highlight text)

**Configuration**
 - OS: MacOS (10.15)
 - Browser: Chrome (83), Firefox (DE 79), Safari (13), Edge (83)

**Additional context**
A small amount of investigation shows that it appears to have something to do with the SVG layer being placed on top of the text layer.  When they are swapped or the text layer's z-index is raised the text becomes selectable.  However then the point(edit) and pen tools don't work (presumably because the SVG layer is now below the text layer).  It may also be important to note that with the text layer on top the comment form no longer auto focuses once you are finished highlighting text (see the [infrastructure demo](http://instructure.github.io/pdf-annotate.js/))

**Comment by MasterOdin** [comment]
Yeah, looking at this, it looks like it was break that was introduced in the long gone DynamicEnvironmentSystems fork (which is what we were principally based off on), where https://github.com/Submitty/pdf-annotate.js/commit/03c8f1da8ba4a61464ecacdeab4df412394d4be5 broke the text selection to I guess improve selection performance and then https://github.com/Submitty/pdf-annotate.js/commit/16255a4bef26b7957153d91c75c8751cfd21aca0 was done by a long since gone undergrad. Reverting both of these fixes (or at least in the case of the first one, flipping annotationLayer and textLayer in the template) the immediate issue here with the text selection / highlighter, and then also making the tools still work, however, not sure what the ramifications might be without further testing.

**Comment by brandscill** [comment]
Are there any plans to resolve this?

**Comment by NicholasCiuica** [comment]
Closing because the issue is no longer relevant. PDF annotation has been removed from Submitty.
